compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/TypeResolver.java

```diff
@@ -441,6 +440,10 @@ private BType calculateEffectiveType(BLangIdentifier name, BLangType typeNode, B
         return potentialIntersectionType;
     }
 
+    private int getTypeTag(BType type) {
```

Comment:
Let's rename this to something like, `getEffectiveTypeTag`.

Reply:
Updated